VKappProject/VKappProject/Controllers/FriendCollectionViewController.swift

```diff
@@ -1,23 +1,31 @@
 // FriendCollectionViewController.swift
 // Copyright © RoadMap. All rights reserved.
 
+import Alamofire
 import UIKit
 
 /// Контроллер с фотографией друга
 final class FriendCollectionViewController: UICollectionViewController {
-    // MARK: - Public Property
+    // MARK: - Public Properties
 
-    var friendAvatarName = ""
+    var id = ""
```

Comment:
```suggestion
    var id = Constants.emptyString
```
Меня заставляли выносить в константы